src/app/users/users-update/users-update.component.ts

```diff
@@ -168,6 +174,11 @@ export class UsersUpdateComponent implements OnInit {
     this.uploadImage = false;
   }
 
+  ResetImage() {
```

Comment:
Methods should start with a lowercase letter.  Only classes should start with an uppercase letter.